.gitignore

```diff
@@ -1,2 +1,2 @@
 /Cargo.lock
-/target/
+**/target/
```

Comment:
Do you regularly get a `target/` subdir created somewhere else with builds in subdirectories?
This might indicate an improperly configured workspace.